pages/index.tsx

```diff
@@ -34,41 +38,131 @@ export default function Home() {
           <link rel="prefetch" href="https://robertbrunhage.com/videos" />
           <link rel="canonical" href="https://robertbrunhage.com" />
         </Head>
-        <div className={styles.app}>
-          <div className={styles.app__main}>
-            <h1>
-              I will <span>soon</span> have flutter courses
-              <span role="img" aria-label="hand pointing down emoji">
-                👇
-              </span>
-            </h1>
-            <p>
-              Future courses will be available here. <br />
-              Sign up now to be notified when released
-              <span role="img" aria-label="smiling emoji">
-                😄
-              </span>
-            </p>
-            <form
-              action="https://gmail.us2.list-manage.com/subscribe/post?u=ff73d806dd2f49da87ede8337&amp;id=ed4e712aca"
-              method="post"
-              name="mc-embedded-subscribe-form"
-              target="_blank"
-              noValidate
-            >
-              <label htmlFor="email">Email</label>
-              <input id="email" type="email" placeholder="johndoe@email.com" name="EMAIL" required />
-              <input id="submit" type="submit" value="Notify me" name="subscribe" />
-              <span>
+        <div className={styles.container}>
```

Comment:
Consider using more html symantec such as `<section></section>` for the different sections.